capstone/src/main/java/com/google/sps/servlets/getNYTDataServlet.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.servlets;
+
+import java.util.*;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import org.json.JSONObject;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/*Servlet that gets COVID count data from NYT*/
+@WebServlet("/getCountyData")
+public class getNYTDataServlet extends HttpServlet {
+  private String NYT_DATA_LINK =
+      "https://raw.githubusercontent.com/nytimes/covid-19-data/master/live/us-counties.csv";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Map<String, Map<String, String>> counties = new HashMap<String, Map<String, String>>();
+    BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+    URL url = new URL(NYT_DATA_LINK);
+    try {
+      HttpURLConnection url_connect = (HttpURLConnection) url.openConnection();
+      url_connect.setRequestMethod("GET");
+      int code = url_connect.getResponseCode();
+      reader = new BufferedReader(new InputStreamReader(url_connect.getInputStream()));
+
+      String input = reader.readLine();
+      Map<String, String> states_visited = new HashMap<String, String>();
+      int states_count = 0;
+      while (input != null) {
+        String parsed_data[] = input.split(",");
```

Comment:
Use a standard serialization format such as JSON instead of comma-separated strings

Reply:
After doing some searching, it appears there are no standardized means of reading .csv files using the Java Standard library, though there is third party libraries out there to do so.